From 4e24dfe76acb7236190aff091245213388957596 Mon Sep 17 00:00:00 2001
From: Zhou Furong <furong.zhou@intel.com>
Date: Mon, 6 May 2019 17:15:56 +0800
Subject: [PATCH 1349/1417] Change keystore applets store path

Change default keystore applet store path to /usr as there is concern
store files under /var

Change-Id: Ic7863ff8b9080e069cd3479e8f6835d57abb9bb6
Signed-off-by: Zhou Furong <furong.zhou@intel.com>
---
 security/keystore/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/keystore/Kconfig b/security/keystore/Kconfig
index e1fb964..e1628eb 100644
--- a/security/keystore/Kconfig
+++ b/security/keystore/Kconfig
@@ -50,7 +50,7 @@ source security/keystore/oemkey/Kconfig
 config DAL_KEYSTORE_APPLET_PATH
 	string "DAL Keystore Applet path"
 	depends on KEYSTORE
-	default "/var/lib/intel/dal/applets/Keystore.pack"
+	default "/usr/lib/dal/applets/Keystore.pack"
 	help
 		The path for DAL Keystore applet file.
 
-- 
2.7.4

